README.md

```diff
@@ -81,6 +81,25 @@ questions or comments.
 
   * pkg/monitor: Monitors running clusters.
 
+  * pkg/operator/controllers: A list of controllers instantiated by the operator
+    component.
+
+    * alertwebhook: Ensures the `alertmanager-main` secret in the
+      `openshift-monitoring` namespace is correct.
```

Comment:
The real story here is that the above secret contains configuration for alertmanager (this might be not straightforward at first, but there is a reason for it: https://github.com/prometheus-operator/prometheus-operator/pull/201)
The operator looks for a "null" receiver in this config and updates it to point to a small webserver in aro-operator which does nothing but replying 200. This is serving as a sink for alert manager, without which customer will get `AlertmanagerReceiversNotConfigured`. 
I think we should add some of the explanation of the "why" rather than translating the code to English "keeping the secret correct" (I know this code was merged before you have joined and this is exactly why we want to have more of this in form of documentation as we have discussed today)

Reply:
Thanks for the feedback. How about:

> Ensures that the receiver endpoint defined in the `alertmanager-main` secret matches the webserver at aro-operator-master.openshift-azure-operator:8080, to avoid the `AlertmanagerReceiversNotConfigured` warning?

Reply:
Great.